### DISCUSSION TITLE
How to set up zfsbootmenu to use separate boot pool?

### DISCUSSION THREAD

**Discussion by mitosu-me** [zbm-dev/zfsbootmenu#314] [Questions] [answered]
I want migrate one of my machine that use grub with separate boot pool, how could redirect zfsbootmenu to read boot pool instead root pool?

**Comment by ahesford** [comment, marked as answer]
You can't. ZFSBootMenu makes few demands on how you arrange your pool(s), but requiring that your `/boot` directory and the kernels therein reside *within* your root filesystem is a fundamental requirement. https://github.com/zbm-dev/zfsbootmenu/blob/master/docs/BOOTENVS.md
